config/routes.rb

```diff
@@ -296,7 +296,11 @@
         :via => :get
   ####
 
+  #### PublicBodyChangeRequest controller
   resource :change_request, :only => [:new, :create], :controller => 'public_body_change_requests'
+  match 'change_request/new/:body' => 'public_body_change_requests#new',
+        :as => :new_change_request_body,
+        :via => :get
```

Comment:
Yes, that's right. Its made `new` and `create` nested actions of the `public_bodies` resource.